targets/install.xml

```diff
@@ -281,8 +281,15 @@
                 <!-- Site-level setup -->
                 <!-- @see https://docs.acquia.com/cloud-platform/performance/memcached/enable/ -->
                 <httpget url="https://docs.acquia.com/_downloads/cloud-memcache-d8+.php" dir="${build.dir}/${drupal.root}/sites/${drupal.site.dir}" filename="settings.acquia-memcache.php"/>
+                <!-- Installing Cloud API SDK v2. -->
+                <composer command="suggest" composer="${composer.composer}">
+                  <arg line="--working-dir ${build.thebuild.dir}" />
+                  <arg value="--list" />
+                  <arg line="| xargs printf -- '%s:^2.0' | xargs composer require --working-dir ${application.startdir}" />
+                </composer>
```

Comment:
Oh this is neat! 

How does this know which suggested packages to install?

🤔 If we have the ability to run composer from a phing task, I wonder if we should just install the package with `composer require` here instead of adding it to the composer.json as `suggest` and running this `suggest` command.  

What do you think?

Reply:
I added this as `suggest` since this was your suggestion, I was having lot of issues with specifying the version though, this is why I ended with this `xargs` solution, I saw there is something similar already when the user specify `platform.sh` as hosting [here](https://github.com/palantirnet/the-build/blob/develop/targets/install.xml#L303-L306). If we want to keep things simpler then this is how it should be.

Reply:
Thanks and sorry for the bad suggestion, I didn't realize we could use composer require in phing but it makes sense that we could!  I'm up for keeping it simple and matching existing patterns. 👍🏼